src/System.Net.Requests/tests/FtpWebRequestTest.cs

```diff
@@ -70,7 +70,7 @@ public void GetResponse_ServerNameNotInDns_ThrowsWebException()
         public void GetResponse_ConnectFailure_ThrowsWebException()
         {
             // This is an invalid IP address, so we should fail to connect.
-            string serverUrl = "ftp://192.0.2.1/";
+            string serverUrl = "ftp://127.0.0.1/";
```

Comment:
I was only responding to the comments above - looking at the bug:
1. There is no guaranteed way to get what you want here - DNS query will always depend on the local network configuration. (i.e. my proposal will take around 1s); using an invalid network address (like `::`) will not test the correct scenario and using a valid random network address may always run the risk of being assigned and having a FTP server or some other system that will not respond to TCP SYN.

2. While Unit Testing will be much faster during execution, a test such like this one is still required to ensure the interactions between our component and the OS.

IMO, the best way to test network is a controlled environment which we don't have at the moment.